cuppa/src/main/java/org/forgerock/cuppa/Cuppa.java

```diff
@@ -211,6 +212,28 @@ public static TestBuilder skip() {
     }
 
     /**
+     * Mark a test or block of tests to be skipped only if the provided {@link Condition} applies.
+     *
+     * <p>The test(s) may still be included in test reports, but marked as skipped.</p>
+     *
+     * <pre><code>
+     * skip(() -&gt; /* ... *&#47;).it("does something", () -&gt; {
+     *   // Will not run if condition evaluates to true.
+     * });
+     * </code></pre>
+     *
+     * @param condition The {@link Condition} that needs to apply in order to skip the test or block of tests.
```

Comment:
Can you add a clarification here about when the condition is executed?

Reply:
"The condition is executed at the time of test definition." ?

Reply:
Sounds good.